### ISSUE TITLE
Map has stopped showing scan results

### ISSUE THREAD

**Issue by Tropicocity** [Mandrakia/Mandrasoft.PokemonGo#14]
Was working until about 20 minutes ago, now it sees no pokémon other than in Paris where it always seems to show them. No issues on login or anything from what I can see, it's just not finding any pokemon :/
![image](https://cloud.githubusercontent.com/assets/20605844/17219781/86b45486-54e4-11e6-8b49-6799da90517b.png)
![image](https://cloud.githubusercontent.com/assets/20605844/17219798/9e532b4e-54e4-11e6-9ca5-7d3391f6597c.png)
![image](https://cloud.githubusercontent.com/assets/20605844/17219828/bb26de82-54e4-11e6-9c22-cf4360a0cbc1.png)

**Comment by Mandrakia** [comment]
Decrease Worker number and retry in 15 min. (put it at 10)
Seems like niantic added a spam detection.

**Comment by Tropicocity** [comment]
Waited a good while, lowered to 10 workers, now getting the following, not looking good :( 
![image](https://cloud.githubusercontent.com/assets/20605844/17220830/f43e1e48-54e8-11e6-975d-733a54652b23.png)

**Comment by luzannew** [comment]
Hmm, there is something going weird.

I do see pokemon at that location. But there are so many pokemon (16000+) that the rendering on the map is so slow and Google Maps crashes...

I think there is something wrong with the import, because when I take a look at that location there are multiple pokemon at the same location (same spawnpoint) with a timestamp in the future.

@Mandrakia Can you check this with this parameters?

neLat:52.44870623523137
neLng:1.7104069729060711
swLat:52.44827384708609
swLng:1.7096519310683789

**Comment by Mandrakia** [comment]
Definitely something goin on here.

I'll check later. Feels like it's got something to do with how we calculate expiration time. I'll need to check.

**Comment by Tropicocity** [comment]
Yeah unfortunately even a couple of hours later I'm still getting nothing :/ It was working perfectly fine earlier today, I was/am so impressed how fast it scanned and how large an area could be covered lol

**Comment by Mandrakia** [comment]
They added some form of throttling. I fixed it locally I'll have to check a few things before comitting/releasing binaries.

**Comment by Tropicocity** [comment]
Any update on this one bud? Your map is legendary!
